pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -100,6 +101,7 @@ stdenv.mkDerivation rec {
     # https://gitlab.com/kicad/code/kicad/-/issues/17133
     "-DCMAKE_CTEST_ARGUMENTS='--exclude-regex;qa_spice'"
   ]
+  ++ optional (!stable) "-DProtobuf_DIR=${protobuf}"
```

Comment:
Is the the expression here just `!stable`, but down below it's `!stable && !testing`?

Should we extract all this into a `let needsProtobuf = ...` at the top of this file?

Reply:
It should be `!stable && !testing`. Should it be extracted to `let needsProtobuf` or `let unstable`?

Reply:
`let needsProtobuf` feels more correct to me. Eventually it should evolve into `let needsProtobuf = true`, and then people will realize they can remove it enitrely. Whereas, `let unstable` feels more likely to stick around forever.

Reply:
@jfly done